website/content/maintainers/community/contributor-growth-framework/motivation.md

```diff
@@ -2,7 +2,6 @@
 title: "Motivating users to contribute"
 linkTitle: "Motivation"
 date: 2020-03-15
-draft: true
 weight: 1
 description: >
```

Comment:
Might be worth adding `kind`/`minkube` to the list of useful tools under `Minimize Steps: Developer Environment Experience`. As that's the big part of validating cloud native tools locally, and should be recognized

Reply:
Thanks for your feedback, @alena1108 

@carolynvs @jberkus, what did we discuss here? I was looking for the YouTube recording of the meeting but can only find videos 7 months ago or older. I think we decided to add a list of tech types vs specific tech. If so, I'll need help here.

Reply:
yeah, I can write up an example, can we move this whole thing to a feature branch so that we can all hack on that?

Reply:
Per our conversation at the last WG meeting, Catherine and I have replaced the table with an explanation of general categories of tools that can be helpful, e.g. linters.